### DISCUSSION TITLE
search and menu not working

### DISCUSSION THREAD

**Discussion by kaciccone** [knowledgefutures/pubpub#3103] [Support]
I am unable to search at https://ncsu-wolfpack-solutions.pubpub.org/search. The search box is unresponsive.

I am also unable to view topics from the drop-down "topics" menu. I can access content through other routes, but this menu item is unresponsive.

**Comment by eastofesten** [comment]
Hi Karen - can you note what browser you're working in? I'm trying to replicate the issue on our end.

**Comment by kaciccone** [comment]
Firefox and iOS. Thanks!sent from my phoneOn Jun 4, 2024, at 12:15 PM, Emily Esten ***@***.***> wrote:﻿
Hi Karen - can you note what browser you're working in? I'm trying to replicate the issue on our end.

**Reply by eastofesten** [reply]
Thanks! Are you able to type in the search box at all? Can you share a video of your experience in Firefox? We're not able to recreate either issue on desktop or mobile.

**Comment by kaciccone** [comment]
When it was happening, I was able to type in the search box, but it
wouldn't search (on my laptop or on my phone). I also could not access the
"topics" drop-down menu on either device. However, now it is working fine.
Mysterious.

On Tue, Jun 4, 2024 at 2:18 PM Emily Esten ***@***.***> wrote:

> Thanks! Are you able to type in the search box at all? Can you share a
> video of your experience in Firefox? We're not able to recreate either
> issue on desktop or mobile.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pubpub/pubpub/discussions/3103#discussioncomment-9668255>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AE32ILWJ26NOJXOAOVRL2RTZFYAGHAVCNFSM6AAAAABIY2DRCGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TMNRYGI2TK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Karen Ciccone
Lead Librarian for Public Science
NC State University Libraries | http://www.lib.ncsu.edu
***@***.***
http://orcid.org/0000-0002-8861-9857

All electronic mail messages in connection with State business which are
sent to or received by this account are subject to the NC Public Records
Law and may be disclosed to third parties.

**Reply by eastofesten** [reply]
Hm, okay. Good to know that it's working currently, and let us know if it happens again!

**Comment by StSchwerdtfeger** [comment]
I had the same issue with dropdown menus twice (they just didn't work anymore and I had to access certain apges via the dashboard). It somehow went away, but today it reappeared. It doesn't matter which browser I am using (checked Firefox and Edge, Win 10). I ran DISM, deleted cookies etc., checked on extensions, so I guess it is not the browser or anything. Last time I also reinstalled firefox, but no change. Last time it depended wether I was logged in or not (no changes in browser incognito modes). This time it makes no difference wether I'm logged in or not. However, when I click on other buttons (like a pub) they appear as if I had marked the title or image. It is this pubpub site: https://journal.medicine.berlinexchange.de/

**Reply by eastofesten** [reply]
Hi Steffen - thanks for letting us know & for trying some additional steps! I replicated both issue on my end in both Chrome & Firefox when not logged-in, but it looks like they've reset for me once the cache was cleared. Do the dropdown menus work for you from this link/now? [https://journal.medicine.berlinexchange.de](https://journal.medicine.berlinexchange.de/?breakCache=1719260618)

**Reply by StSchwerdtfeger** [reply]
Now dropdown menus are working again! Thanks so much for the fast response!
